mitx/lms/templates/courseware/course_about.html

```diff
@@ -1,17 +1,17 @@
 <%page expression_filter="h" />
 <%namespace name='static' file='../static_content.html'/>
 <%!
-import math
-from datetime import datetime, timedelta
+from math import ceil
 from django.utils.translation import ugettext as _
 from django.utils.translation import pgettext
 from django.urls import reverse
-from courseware.courses import get_course_about_section
+from lms.djangoapps.courseware.courses import get_course_about_section
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
```

Comment:
Normally, in openedx we sort the imports in this fashion
1 system-level imports like `import io`
`#` blank line
2 installed packages imports like `from django.conftrib.auth.models import User`
`#` blank line
3 project-level imports like `from openedx.core.djangoapps.content.course_overviews.models import CourseOverview`